src/main/java/ua/shpp/eqbot/telegrambot/EqTelegramBot.java

```diff
@@ -1,32 +1,90 @@
 package ua.shpp.eqbot.telegrambot;
 
+import org.modelmapper.ModelMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 import org.telegram.telegrambots.bots.TelegramLongPollingBot;
 import org.telegram.telegrambots.meta.api.objects.Update;
+import ua.shpp.eqbot.command.CommandContainer;
+import ua.shpp.eqbot.model.UserDto;
+import ua.shpp.eqbot.model.UserEntity;
+import ua.shpp.eqbot.repository.UserRepository;
+import ua.shpp.eqbot.service.SendBotMessageServiceImpl;
 
-import ua.shpp.eqbot.processors.Processor;
+import java.util.HashMap;
+import java.util.Map;
+
+import static ua.shpp.eqbot.command.CommandName.NO;
 
 @Component
 public class EqTelegramBot extends TelegramLongPollingBot {
+    private final static Logger LOGGER = LoggerFactory.getLogger(EqTelegramBot.class);
+    boolean isNewUser = true;
+    private final Map<Long, UserDto> users_active_session = new HashMap<>();
+    public static String COMMAND_PREFIX = "/";
+    private final CommandContainer commandContainer;
+    /*TODO repository here ?*/
+    private final UserRepository userRepository;
+    private final ModelMapper modelMapper = new ModelMapper();
+
+
+    @Autowired
+    public EqTelegramBot(UserRepository userRepository) {
+        this.userRepository = userRepository;
+        this.commandContainer = new CommandContainer(new SendBotMessageServiceImpl(this), userRepository);
+    }
+
 
     @Value("${telegram.bot.name}")
     private String botUsername;
 
     @Value("${telegram.bot.token}")
     private String botToken;
 
-    @Autowired
-    public void setProcessor(Processor processor) {
-        this.processor = processor;
-    }
-
-    private Processor processor;
-
     @Override
     public void onUpdateReceived(Update update) {
-        processor.process(update);
+        if (update.hasMessage() && update.getMessage().hasText()) {
```

Comment:
Is it possible to somehow take the logic to a separate class?

Reply:
it goes beyond my task, it is being completed by Vitaly